python/packages/autogen-ext/src/autogen_ext/models/openai/_openai_client.py

```diff
@@ -784,7 +786,10 @@ async def create_stream(
                 # NOTE: for OpenAI, tool_calls and content are mutually exclusive it seems, so we can skip the rest of the loop.
                 # However, this may not be the case for other APIs -- we should expect this may need to be updated.
                 continue
-
+            #thought: str | None = None
+            if choice.delta.model_extra['reasoning_content'] is not None:
```

Comment:
Same as before, we need to make sure the field exists and the key "reasoning_content" exists.